#!/bin/sh /usr/share/dpatch/dpatch-run 
## 50-kolabpasswd-slappasswd-path by Noèl Köthe <noel@debian.org>
##
## DP: This is the patch to correct the slappasswd path

@DPATCH@
--- kolabd-1.9.4/kolabpasswd.orig	2006-01-26 13:18:12.000000000 +0100
+++ kolabd-1.9.4/kolabpasswd	2006-01-26 13:18:29.000000000 +0100
@@ -37,7 +37,7 @@
 # Hash a password
 sub hashPassword {
   my $pw = shift;
-  my $hashcmd = $kolab_prefix."/sbin/slappasswd -s ".qquote($pw,":shell");
+  my $hashcmd = "/usr/sbin/slappasswd -s ".qquote($pw,":shell");
   (my $hashpw = `$hashcmd`) or die $@;
   chomp($hashpw);
   return $hashpw;
